### PR TITLE
Improves admin counting for irc

### PR DESCRIPTION
**Pull request by tkdrg**
This is intended to support future improvements to the #adminbus bot.

### CODE REVIEW

**Comment by Shadowlight213** [comment]
:+1:

**Comment by tkdrg** [comment]
Apparently world/Topic() is accessible remotely so giving away stealthmins here is a bad idea. Should we disclose the amount of AFK admins, though? It doesn't look like it's that big of a deal to me.

**Comment by Shadowlight213** [comment]
A better solution would probably be to use the server bots themselves for the input and output commands. Would bypass the security issue of topic.

File: code/world.dm [hunk 1]
```diff
@@ -96,23 +96,20 @@ var/global/list/map_transition_config = MAP_TRANSITION_CONFIG
...
+
+		var/list/adm = get_admin_counts()
+		s["admins"] = adm["present"]
 		s["gamestate"] = 1
 		if(ticker)
 			s["gamestate"] = ticker.current_state
 		s["map_name"] = map_name ? map_name : "Unknown"
+		s["noflagadmins"] = adm["noflags"]
+		s["afkadmins"] = adm["afk"]
+		s["stealthadmins"] = adm["stealth"]
```
Comment (MrStonedOne) [inline code/world.dm]: anybody can view this info, adding stealthmins isn't a wise idea.

**Comment by tkdrg** [comment]
Alright, it will now use send2irc for the secret info (of course, this will require matching changes to the bot, which are planned).

**Comment by MrStonedOne** [comment]
actually, if you want to be lazy, look at pr announce code, it has a coms key, you could change status to support giving that key for giving more info.

**Comment by tkdrg** [comment]
Is this current solution also acceptable? Implementing the comms key sounds like a little bit more effort.

**Comment by MrStonedOne** [comment]
not really, it's rather easy, i'll pr it to you

**Comment by Shadowlight213** [comment]
@tkdrg @MrStonedOne rip?

**Comment by tkdrg** [comment]
this has been done for a while but it's pointless without the corresponding updates to N-bot which I have sent to mso

**Comment by tkdrg** [comment]
This should be fine I think, assuming @MrStonedOne got my patch to run.

**Comment by Cheridan** [comment]
if you say so!

h-here i go

**Pull request merged**